scripts/delete-service.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+
+# Delete dualstack service
+oc delete  service/test-service-ipv6 -n "${TNF_EXAMPLE_CNF_NAMESPACE}" --ignore-not-found=true
```

Comment:
Could this just point to the file with `-f` instead of calling out the service by name?

Reply:
fixed, good idea!